src/TabBar.tsx

```diff
@@ -136,6 +138,10 @@ const TabBar: CustomTabBarType = forwardRef(function TabBar(props, ref) {
 
     if (scrollUpIfNeeded(activePageRef?.current)) return;
 
+    // if the profile page is already open, show the account switcher
+    if (router.getRouteInfo()?.pathname === "/profile")
```

Comment:
`location.pathname` instead to be consistent with the rest of the component :)

Reply:
No problem